functions/menu.bash

```diff
@@ -183,7 +183,7 @@ show_main_menu() {
     "42 | Upgrade to openHAB 3"   "Upgrade OS environment to openHAB 3 release" \
     "   | Downgrade to openHAB 2" "Downgrade OS environment from openHAB 3 back to openHAB 2 stable (DANGEROUS)" \
     "43 | Remote Console"         "Bind the openHAB SSH console to all external interfaces" \
-    "44 | Reverse Proxy"          "Setup Nginx with password authentication and/or HTTPS access" \
+    "44 | Nginx Proxy"            "Setup reverse and forward web proxy" \
```

Comment:
I actually really like this change in wording I never could figure out a good way to word it but I think this works well.

Reply:
thanks. yeah I have been thinking the same in a long time.